src/ssl/ErrorDetailManager.h

```diff
@@ -84,7 +82,7 @@ class ErrorDetailsManager
     /// cache the given error details list.
     void cacheDetails(const ErrorDetailsList::Pointer &errorDetails) const;
 
-    typedef std::map<std::string, ErrorDetailsList::Pointer> Cache;
+    typedef std::map<SBuf, ErrorDetailsList::Pointer> Cache;
```

Comment:
We try to upgrade typedefs when changing them:

```suggestion
    using Cache = std::map<SBuf, ErrorDetailsList::Pointer>;
```